src/state/megaStore.tsx

```diff
@@ -459,6 +465,14 @@ export const Provider: ParentComponent = (props) => {
         setInterval(async () => {
             await actions.sync();
```

Comment:
i think actions.sync() gets the price too, should do a sweep to see where we are fetching the price

Reply:
this pr updates sync() to not do price anymore